readme.txt

```diff
@@ -1,8 +1,8 @@
 === New Relic Reporting for WordPress ===
 Contributors:      10up, rittesh.patel, tott, oscarssanchez, jeffpaul
 Tags:              New Relic, New Relic Reporting, New Relic APM Reporting, APM Reporting
-Requires at least: 4.0
-Tested up to:      6.3
+Requires at least: 6.3
+Tested up to:      6.5.2
 Stable tag:        1.3.2
 Requires PHP:      7.3.11
```

Comment:
Can we bump to 7.4 or 8.0 here as well?

Reply:
Updated this here: https://github.com/10up/wp-newrelic/pull/66/commits/5d1c7bb716fadb3398d325f0efcbee104cf2a149